integration-tests/src/test/java/io/apicurio/tests/migration/DataMigrationIT.java

```diff
@@ -83,8 +85,8 @@ public static class MigrateTestInitializer extends AbstractTestDataInitializer {
 
         @Override
         public Map<String, String> start() {
-
-            String registryBaseUrl = startRegistryApplication("quay.io/apicurio/apicurio-registry-mem:2.4.14.Final");
+            // TODO we will need to change this to 3.0.0 whenever that is released!
```

Comment:
Can you open an Issue for this?

Reply:
https://github.com/Apicurio/apicurio-registry/issues/4087